### ISSUE TITLE
Can load vibs Error

### ISSUE THREAD

**Issue by FoldZero** [itiligent/ESXi-Custom-ISO#8]
### New error running script: 

```
PS D:\Apps\Vmware\ESXi-Custom-ISO> .\esxi6.7.ps1

Retrieving latest ESXi 6.7 bundle, this may take a while...

WARNING: Please consider joining th...

Add-EsxSoftwareDepot : 

  **Failed to initialize the VMware.ImageBuilder PowerCLI module because of error: Failed to load VibSign Python module. 

Make sure OpenSSL 1.1 is installed.**
Make sure that Python 3.7.1 or higher is installed and that you have set the path to the Python executable by using
Set-PowerCLIConfiguration -PythonPath <executable_path>. See the PowerCLI Compatibility Matrixes for information on
the Python requirements.
At D:\Apps\Vmware\ESXi-Custom-ISO\esxi6.7.ps1:33 char:1
+ Add-EsxSoftwareDepot https://hostupdate.vmware.com/software/VUM/PRODU ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-EsxSoftwareDepot], Exception
    + FullyQualifiedErrorId : System.Exception,VMware.ImageBuilder.Commands.AddDepot
```

### My Version - 

```
PS C:\WINDOWS\system32> pip list
Package      Version
------------ -------
cffi         1.15.1
cryptography 42.0.7
lxml         5.2.2
pip          24.0
psutil       5.9.8
pycparser    2.21
pyOpenSSL    24.1.0
setuptools   47.1.0
six          1.16.0
wheel        0.42.0
```

### Python version 
`py : Python 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:01:55) [MSC v.1900 32 bit (Intel)] on win32`

### My power.cli
```
PS D:\Apps\Vmware\ESXi-Custom-ISO> Get-Module VMware.PowerCLI -ListAvailable

    Directory: C:\Program Files\WindowsPowerShell\Modules

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   13.1.0.... VMware.PowerCLI


PS D:\Apps\Vmware\ESXi-Custom-ISO> (Get-Module VMware.PowerCLI -ListAvailable).RequiredModules

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     13.1.0.... VMware.VimAutomation.Sdk
Script     13.1.0.... VMware.VimAutomation.Common
Script     8.1.0.2... VMware.Vim
Script     13.1.0.... VMware.VimAutomation.Core
Script     12.7.0.... VMware.VimAutomation.Srm
Script     12.0.0.... VMware.VimAutomation.License
Script     13.1.0.... VMware.VimAutomation.Vds
Script     12.6.0.... VMware.CloudServices
Script     13.0.0.... VMware.VimAutomation.Vmc
Script     13.1.0.... VMware.VimAutomation.Nsxt
Script     13.1.0.... VMware.VimAutomation.vROps
Script     13.1.0.... VMware.VimAutomation.Cis.Core
Script     13.1.0.... VMware.VimAutomation.HorizonView
Script     13.1.0.... VMware.VimAutomation.Cloud
Script     8.0.0.2... VMware.DeployAutomation
Script     8.0.0.2... VMware.ImageBuilder
Script     13.1.0.... VMware.VimAutomation.Storage
Script     1.6.0.0    VMware.VimAutomation.StorageUtility
Script     12.7.0.... VMware.VumAutomation
Script     13.1.0.... VMware.VimAutomation.Security
Script     13.0.0.... VMware.VimAutomation.Hcx
Script     12.4.0.... VMware.VimAutomation.WorkloadMan...
Script     1.0.111... VMware.Sdk.Runtime
Script     8.0.111... VMware.Sdk.vSphere
Script     12.6.0.... VMware.PowerCLI.VCenter
Script     4.1.0.2... VMware.Sdk.Nsx.Policy
Script     8.7.0.2... VMware.Sdk.Srm
Script     8.7.0.2... VMware.Sdk.Vr
```

**Comment by FoldZero** [comment]
Now Resolved using Power CLI 12.5 - https://developer.broadcom.com/tools/vmware-powercli/12.5.0****

**Comment by itiligent** [comment]
Thanks for your info. I was not able to reproduce your error on Win10x64.  

As you seem to be using 32bit Python which I have not tested,  there may be some 32bit issues there with various vib digital signatures or encryption and such. I cant be sure.

Your post did however give me the idea to update the Readme to recommend using PowerCLI 12.7.0 just for ESXi 6.x ISOs (old school & no python requirement) I've tested 12.7.0 for the final  6.7 build this works fine. 

Users will need to stay with latest PowerCLI and Python 3.7.x for 7.x and 8.x ISOs.
